parsec/backend/memory/organization.py

```diff
@@ -170,6 +171,58 @@ async def stats(self, id: OrganizationID) -> OrganizationStats:
             users_per_profile_detail=users_per_profile_detail,
         )
 
+    async def server_stats(self, from_date: DateTime, to_date: DateTime = DateTime.now()):
```

Comment:
Return type should be typed, and it would be much better to return a class instance instead of a dict (this makes typing happy and typo errors in dict is a plague ^^)

see what is done for `OrganizationStats`